x-pack/platform/test/alerting_api_integration/spaces_only/tests/alerting/group4/alerts_as_data/install_resources.ts

```diff
@@ -150,10 +150,16 @@ export default function createAlertsAsDataInstallResourcesTest({ getService }: F
           '.internal.alerts-test.patternfiring.alerts-default-*',
           '.reindexed-v8-internal.alerts-test.patternfiring.alerts-default-*',
         ]);
-        expect(contextIndexTemplate.index_template.composed_of).to.eql([
-          '.alerts-test.patternfiring.alerts-mappings',
-          '.alerts-framework-mappings',
-        ]);
+
+        expect(
+          contextIndexTemplate.index_template.composed_of.includes(
+            '.alerts-test.patternfiring.alerts-mappings'
+          )
+        );
+        expect(
+          contextIndexTemplate.index_template.composed_of.includes('.alerts-framework-mappings')
+        );
```

Comment:
This test now accounts for `composed_of` possibly including the `fast_refresh` component template